Source/VisualStudioTools/Private/VSTestAdapterCommandlet.cpp

```diff
@@ -165,16 +168,16 @@ UVSTestAdapterCommandlet::UVSTestAdapterCommandlet()
 	HelpUsage = TEXT("<Editor-Cmd.exe> <path_to_uproject> -run=VSTestAdapter [-stdout -multiprocess -silent -unattended -AllowStdOutLogVerbosity -NoShaderCompile]");
 
 	HelpParamNames.Add(ListTestsParam);
-	HelpParamDescriptions.Add(TEXT("[Optional] The file path to write the test cases retrieved from FAutomationTestFramework"));
+	HelpParamDescriptions.Add(TEXT("[Required] The file path to write the test cases retrieved from FAutomationTestFramework"));
 
 	HelpParamNames.Add(RunTestsParam);
-	HelpParamDescriptions.Add(TEXT("[Optional] The test cases that will be sent to FAutomationTestFramework to run."));
+	HelpParamDescriptions.Add(TEXT("[Required] The test cases that will be sent to FAutomationTestFramework to run."));
 
 	HelpParamNames.Add(TestResultsFileParam);
-	HelpParamDescriptions.Add(TEXT("[Optional] The output file from running test cases that we parse to retrieve test case results."));
+	HelpParamDescriptions.Add(TEXT("[Required] The output file from running test cases that we parse to retrieve test case results."));
 
-    HelpParamNames.Add(FiltersParam);
-    HelpParamDescriptions.Add(TEXT("[Optional] List of test filters to enable separated by '+'. Default is 'smoke+product+perf+stress+negative'"));
+	HelpParamNames.Add(FiltersParam);
+	HelpParamDescriptions.Add(TEXT("[Required] List of test filters to enable separated by '+'. Default is 'smoke+product+perf+stress+negative'"));
```

Comment:
This one isn't required. If -filters isn't passed in we just use the defaults

Reply:
Sounds good